e2e/src/tests/chatHeader.test.ts

```diff
@@ -179,7 +181,8 @@ test(
       await dialHomePage.acceptBrowserDialog(
         ExpectedConstants.clearAllConversationsAlert,
       );
-      await chatHeader.clearConversation.click();
```

Comment:
please, do not delete click on clear btn line;
instead, you should delete browser dialog handling in lines 171 and 181.

Also, you should add dialog cancel dialog action for step 'Try to clear conversation messages using header button but cancel clearing and verify no messages deleted' so that it looks like this

`await test.step('Try to clear conversation messages using header button but cancel clearing and verify no messages deleted', async () => {
      await dialHomePage.openHomePage();
      await dialHomePage.waitForPageLoaded();
      await chatHeader.clearConversation.click();
      await confirmationDialog.cancelDialog();

      const messagesCount = await chatMessages.chatMessages.getElementsCount();
      expect
        .soft(messagesCount, ExpectedMessages.messageContentIsValid)
        .toBe(conversation.messages.length);
    });`

`await test.step('Clear conversation messages using header button and verify messages deleted, setting are shown', async () => {
      await chatHeader.clearConversation.click();
      await confirmationDialog.confirm();

      const isConversationSettingsVisible =
        await conversationSettings.isVisible();
      expect
        .soft(
          isConversationSettingsVisible,
          ExpectedMessages.conversationSettingsVisible,
        )
        .toBeTruthy();
    });`